### DISCUSSION TITLE
Multi model mark

### DISCUSSION THREAD

**Discussion by alexsalramxr** [maize-tech/laravel-markable#14] [Q&A] [answered]
Hello, I want to know about that package, because is the first time that I use it.
My question is, allows to other models (e.g. Customer) mark any other model as favorite, like or bookmark?
I see the config file and only with user model is possible.

**Comment by riccardodallavia** [comment, marked as answer]
Hi @alejandrosramirez and thanks for your suggestion!
We had already thought about the possibility of supporting multiple entities, but this would have required introducing a polymorphic relationship, overcomplicating the logic of the package with respect to our use cases.
If you have any ideas to implement the feature without impacting the performances, please feel free to submit a pull request!
Thank you!
